index.html

```diff
@@ -32,7 +32,7 @@
   <div id="cspace"></div>
   <script>
     cspaceUI({
-      serverUrl: 'http://localhost:8180',
+      serverUrl: 'https://botgarden-dev.cspace.berkeley.edu',
```

Comment:
hmm... is that really a .... robust .... way of coding that? how will it get updated for production? ;-)

Reply:
Had to change it to be able to test on dev, but I fixed it right away, hence the "Outdated"